### PR TITLE
SkyDNS Updates

### PR DESCRIPTION
**Pull request by harsha-y**
- update skydns image to the version released for k8s 1.1.8
- update skydns rc yaml to reflect latest health check pods

https://github.com/kubernetes/kubernetes/blob/v1.1.8/cluster/addons/dns/skydns-rc.yaml.in

### CODE REVIEW

File: plugins/dns/dns-service.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ metadata:
 spec:
   selector:
     k8s-app: kube-dns
-  clusterIP:  10.100.0.10
+  clusterIP: 10.100.0.10
```
Comment (harsha-y) [inline plugins/dns/dns-service.yaml]: removing unnecessary whitespace

**Comment by pires** [comment]
Thank you @harsha-y :)
